packages/babel-generator/src/generators/expressions.js

```diff
@@ -92,6 +92,7 @@ export function Decorator(node: Object) {
 
 export function CallExpression(node: Object) {
   this.print(node.callee, node);
+  if (node.loc) this.printAuxAfterComment();
```

Comment:
This forces any after comments to be output prior to the `(` in a call. The [diff below](https://github.com/babel/babel/pull/3183/files#diff-ddede16cd4c0d669d1659ed1ab4cf64eL29) has test coverage for this:

```
-/*before*/(0, _foo4. /*after*/bar)( /*before*/_foo2.default) /*after*/;
+/*before*/(0, _foo4.bar) /*after*/( /*before*/_foo2.default /*after*/, /*before*/_foo5.foo /*after*/);
```

This is a bit of whack-a-mole and is not a complete fix for the handling of aux after but helps fix one of the identified cases.